docs/Modules/common-modules/event-bus.md

```diff
@@ -9,4 +9,146 @@ parent: "Common Modules"
+
+## on
+
+### Usage
+
+on(events: _String_ or _Array< string >_, [priority=1000]: _number_, callback: _Function_, [that]:_Object_): _undefined_
+
+### Parameters
+
+* `events`: name of the event or _Array_ of event names.
+* `priority`: optional parameter specifying the priority in which this listener is called, where a higher number represents a higher priority and therefore leads to earlier being called. If not provided, default of `1000` will be used.
+* `callback`: function which will be invoked with `event, ...additionalArguments`. Returning anything but `undefined` from this will stop the listener propagation. Returning `false` will prevent the execution of the default action for that event.
+*  `that`: optional parameter which can be used to pass context (`this`) to the callback.
+
+### Return Value
+
+_undefined_
+
+### Code Example
+```javascript
+eventBus.on('saveSVG.start', 1500, function(event) {
+  // access event type
+  event.type; // 'saveSVG.start'
+
+  // stop propagation to other listeners
+  event.stopPropagation();
+
+  // prevent event default
+  event.preventDefault();
+
+  // Some custom logic
+  // [...]
+});
+```
+
+
+## once
+
+### Usage
+
+once(events: _String_ or _Array< string >_, [priority=1000]: _number_, callback: _Function_, [that]:_Object_): _undefined_
+
+### Parameters
+
+* `events`: name of the event or _Array_ of event names.
+* `priority`: optional parameter specifying the priority in which this listener is called, where a higher number represents a higher priority and therefore leads to earlier being called. If not provided, default of `1000` will be used.
+* `callback`: function which will be invoked with `event, ...additionalArguments`. Returning anything but `undefined` from this will stop the listener propagation. Returning `false` will prevent the execution of the default action for that event.
+*  `that`: optional parameter which can be used to pass context (`this`) to the callback.
+
+### Return Value
+
+_undefined_
+
+## off
+
+### Usage
+
+off(events: _String_ or _Array< string >_, [callback]: _Function_): _undefined_
+
+### Parameters
+
+* `events`: name of the event or _Array_ of event names.
+* `callback`: optional parameter specifying the callback listener to be removed.
+
+### Return Value
+
+_undefined_
+
+### Code Example
+```javascript
+function listener(event) {
+  // access event type
+  event.type; // 'saveSVG.start'
+
+  // stop propagation to other listeners
+  event.stopPropagation();
+
+  // prevent event default
+  event.preventDefault();
+}
+
+eventBus.on('saveSVG.start', 1500, listener);
+eventBus.off('saveSVG.start', listener);
+```
+
+## createEvent
+
+### Usage
+
+createEvent(data: _Object_): _Object_
+
+### Parameters
+
+* `data`: the event data.
+
+### Return Value
+
+An _InternalEvent_ object which can be passed to [fire](#fire).
+
+## fire
+
+### Usage
+
+fire(name: _String_, [data]: _Object_, [arguments]: _...Object_): _Object_
+
+### Parameters
```

Comment:
we should make it consistent whether to use an article ("the"), e.g.

> the event name.

Or leave it without, e.g.

> optional additional arguments ...

Reply:
I agree, let's have a separate discussion about this covering all content that we have so far